scripts/ngless-cwl/helpers/make_cwl.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# make_cwl.sh -- created 2017-03-09 - Renato Alves
+
+set -e
+
+echo "This script is used to create a template .cwl file from existing scripts"
+echo "the final result will be incomplete. You'll need to manually adjust"
+echo "output: and type: File for inputs:"
```

Comment:
I thought that using `argparse.FileType` got converted into CWL's `type: File` correctly?
https://docs.python.org/2.7/library/argparse.html#filetype-objects